.github/workflows/tests.yml

```diff
@@ -18,11 +18,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.7.x", "2.6.x", "2.5.x"]
+        ruby: ["3.0.x", "2.7.x", "2.6.x", "2.5.x"]
         gemfile:
           - Gemfile
-          - gemfiles/rails5.gemfile
-          - gemfiles/rails6.gemfile
+          - gemfiles/rails6_1.gemfile
+          - gemfiles/rails6_0.gemfile
```

Comment:
Does this change mean that Rails 5 support was dropped? Or is support for Rails 5.2 tested in any way?